digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -682,6 +685,45 @@ protected void setEcsTaskOverride(
         request.withOverrides(taskOverride);
     }
 
+    protected void validateTaskOverride(
+            final CommandContext context,
+            final CommandRequest request,
+            final ContainerDefinition cd)
+            throws ConfigException
+    {
+        // Note: A total of 8192 characters are allowed for overrides. This limit includes
+        // the JSON formatting characters of the override structure.
+        //
+        // see https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RunTask.html#API_RunTask_RequestSyntax
+        //
+        // This is simplified validation. The other payload size to environment variables is not exactly 192.
+        int total = 0;
+        for (Map.Entry<String, String> e : request.getEnvironments().entrySet()) {
+            String k = e.getKey();
+            String v = e.getValue();
```

Comment:
Is a nested JSON object represented as a JSON string here, right?

Reply:
I assume so. 

setEcsContainerOverrideEnvironment would make the following json for API request.

```
   "overrides": { 
      "containerOverrides": [ 
         { 
            "command": [ "string" ],
            "cpu": number,
            "environment": [ 
               { 
                  "name": "string",
                  "value": "string"
               },
               { 
                  "name": "string",
                  "value": "string"
               },
              ...
            ],
 ```